resources/recruiting-board/board-responsibilities.md

```diff
@@ -0,0 +1,60 @@
+
+The marketing committee is composed of volunteers that are primarily marketers, not developers (although there are some more technical than others). They are developer marketing and technical marketers that care about and/or have bet their businesses on .NET. This is a smaller committee because most people that show interest in volunteering are not interested in marketing and are referred to the outreach committee which is focused on community activities and education.
+
+The main goals of the marketing committee are:
+
+- Increase membership
+- Generate awareness (of .NET open source and the .NET Foundation itself)
+- Foster and grow an open, healthy.NET community
+
+#### Marketing Committee Chair
+
+The marketing committee chair is responsible for directing the committee’s marketing strategy and execution.
+
+**Always-on activities** The chair runs monthly meetings with members and is on point for all announcements and air cover (for dealing with bad PR if needed as it arises). They are responsible for the foundation’s web presence, running all the social handles(Twitter, Facebook, LinkedIn) and growing followers, pulling together and sending the bi-monthly newsletter(MailChimp), driving leads(for newsletter and membership), running the [SWAG store](https://store.dotnetfoundation.org), writing and sending press releases.
+
+**Other activities** They are also responsible for new project “go-to-market” GTM, co-marketing initiatives, event sponsorships & presence, some event management (i.e. .NET Conf), website design & copy,branding and logos, surveys, presentation decks, and coordination with other committees around messaging and awareness as needed.
+
+***Time commitment: 40-60 hrs./month***
+
+## [Outreach Committee](https://github.com/dotnet-foundation/wg-outreach)
+
+The Outreach Committee is here to welcome developers of all backgrounds,education, and technology ecosystems into the .NET ecosystem. We are here to help you organize and evangelize.
+
+High-level goals:
+
+- Encourage new developers to build with .NET
+- Empower underrepresented segments of the coder community to become leaders and contributors
+- Assist event organizers with evangelism and growth
+
+## [Membership Committee](https://github.com/dotnet-foundation/wg-membership)
+
+The committee is responsible for evaluating new member applications and discussing membership criteria. The goal is to grow the membership in the foundation, and ensure that members feel that foundation membership provides value. This committee works to create new initiatives that encourage our membership to participate more in the foundation’s activities. These encompass many different roles, from open source projects to career development and mentoring to education opportunities.
```

Comment:
Does it need the time commitment outlined like in the Marketing Committee Chair section?

And should all these sections have the same format

- Overview 
- Main goals in bullet points
- Time Commitment